packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -35,6 +36,19 @@ import { PluginScmProvider, PluginScmResourceGroup, PluginScmResource } from '..
 import { ResourceContextKey } from '@theia/core/lib/browser/resource-context-key';
 import { PluginViewWidget } from '../view/plugin-view-widget';
 import { ViewContextKeyService } from '../view/view-context-key-service';
+import { WebviewWidget } from '../webview/webview';
+import { Navigatable } from '@theia/core/lib/browser/navigatable';
+
+type CodeEditorWidget = EditorWidget | WebviewWidget;
+export namespace CodeEditorWidget {
```

Comment:
I'm not sure this is the appropriate place for this type and namespace.
I don't expect `menus-contribution-handler` to have this information.

Reply:
I copy the code from #5527 @akosyakov

Reply:
@vince-fugnitto Where would you suggest to put it? It is require for now only for menu contributions to provide mapping between vscode core editor meaning to Theia.